### ISSUE TITLE
Turbowarp support?

### ISSUE THREAD

**Issue by AlexSpaces** [MeechaPooch/Blocklive#18]
i have been trying to do a complex **scratch** project with my group and was using **blocklive** before when on base **scratch**, when i try to do complex projects i'm usually working solo but now when i started thinking about it i couldn't use **blocklive** on **turbowarp**.
to be completely honest, i'm just asking to either see if its something your working on or to give you an idea :)

**Comment by BroJac5246** [comment]
I could've sworn there's an open PR to add this to Scratch Addons, which TurboWarp uses for extensions, but I couldn't find it so maybe it doesn't exist(?)

I'm not the developer of Blocklive, but adding support for a new website would probably be very time-consuming. There's also the issue of figuring out how to authenticate users using TurboWarp, given that it doesn't have accounts that you can enter to say "I want to work with this user." It would be cool to see support, but ultimately it's probably too much effort for relatively little benefit. I like the idea, though.

**Comment by AlexSpaces** [comment]
There's also the issue of figuring out how to authenticate users using TurboWarp, given that it doesn't have accounts that you can enter to say "I want to work with this user.


Ok well then what about [Penguin Mod](https://penguinmod.com), it uses scratch accounts to sign in and has turbowarp built in

**Comment by BroJac5246** [comment]
> Ok well then what about Penguin Mod, it uses scratch accounts to sign in and has turbowarp built in

It's not compatible with Scratch, right? Has someone actually suggested to them that they add their own version of this?

**Comment by AlexSpaces** [comment]
@BroJac5246 i don't think so, I suggested penguin mod because it's  able to authenticate (using scratch) but that still doesn't fix the other issues 

[video here](https://github.com/MeechaPooch/Blocklive/assets/103975358/ca4df1c7-9546-4f55-be78-bc416e3fdc0e)
